deployment/cloud-run/secure_api_server.py

```
@@ -22,21 +23,165 @@
+def load_all_models():
+    """Consolidated model loading function for all AI models"""
+    global t5_summarizer, whisper_transcriber, T5_AVAILABLE, WHISPER_AVAILABLE
+
+    logger.info("🔄 Loading all AI models...")
+
+    # Load emotion detection model
+    try:
+        load_model()
+        logger.info("✅ Emotion detection model loaded")
+    except Exception as e:
+        logger.error(f"❌ Failed to load emotion detection model: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)